### PR TITLE
Completed css-brochure-form | Chris Marron

### PR DESCRIPTION
**Pull request by chris-marron**
image

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+form {
+  width: 50%;
+  margin: auto;
+}
+
+label {
+  margin-left: 15px;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:20]: This line is not necessary here, it can be deleted.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+h3 {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:5]: This line is not necessary here, it can be deleted.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-brochure-form/styles.css (hunk 1 above)
Comment (chris-marron) [reply css-brochure-form/styles.css]: done.

File: css-brochure-form/styles.css (hunk 2 above)
Comment (chris-marron) [reply css-brochure-form/styles.css]: > 

changed the buttonimage

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**